### DISCUSSION TITLE
Word holder op med at svare ved kopiering og beregning

### DISCUSSION THREAD

**Discussion by clarafoghgym** [Eduap-com/WordMat#142] [Support]
**Windows eller Mac**
Jeg har Windows 11

**WordMat version**
Nyeste version - 1.26

**Beskriv fejlen**
Problemet er at Word holder op med at svare når jeg foretager helt simple ting i ligningsfelter og med Wordmat. Dette varer ved i et halvt minut til 5 min og det virker derefter indtil jeg igen gør en af de ting der udløser problemet. Fejlen opstår når jeg i et matematikfelt forsøger at beregne noget, løse en ligning eller kopiere noget indhold. Jeg ved ikke om fejlen opstår ved brug af andre funktioner da jeg ikke anvender andet. Det skal siges at jeg kun anvender tastaturgenveje og derfor ikke ved om det også sker hvis knapperne på båndet bruges.

Jeg har flere gange forsøgt at genstarte Wordmat, at afinstallere og geninstaller programmet, at genstarte computeren og at opdatere Word og Windows. Dette har ikke gjort nogen forskel.

**Trin til genskabelse af fejlen**

1. Lav matematikfelt
2. Skriv udtryk (hvad som helst der skal udregnes)
3. Beregn ved AltGr + Enter eller Alt + B

eller

1. Lav matematikfelt
2. Skriv ligning
3. Løs ligning ved Alt + L

eller

1. Lav matematikfelt
2. Skriv noget
3. Marker en del af det
4. Kopier ved Ctrl + C (her opstår fejlen nogle gange)
5. Indsæt i matematikfeltet ved Ctrl + V (her opstår fejlen også)

**Forventet opførsel**
Jeg forventede at Word ikke gik i stå.

**Dokument**
Dokument der kan frembringe fejlen:
[Lommeregner dokument.docx](https://github.com/Eduap-com/WordMat/files/12570175/Lommeregner.dokument.docx)
Dette dokument er bare et dokument der kan frembringe fejlen. Alle nye og gamle gør det også.

**Screenshots**

Jeg har her bedt om en beregning:
![Screenshot 2023-09-10 221742](https://github.com/Eduap-com/WordMat/assets/144626515/89dae224-4916-44fe-8cfa-3047d09f46c9)
Her er svarer Word igen og der er fremkommet et resultat:
![Screenshot 2023-09-10 221824](https://github.com/Eduap-com/WordMat/assets/144626515/70d5947f-8fd3-4c42-9501-a4574af85e07)

**Comment by Eduap-com** [comment]
Problemet er computerspecifikt og vil kræve support. Overføres til diskussioner
